functests/1_performance/performance.go

```diff
@@ -151,6 +151,15 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				Expect(string(initrd)).ShouldNot(ContainSubstring("'/etc/systemd/system.conf /etc/systemd/system.conf.d/setAffinity.conf'"))
 			}
 		})
+
+		It("[test_id:][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running on the host", func() {
```

Comment:
@gsr-shanks  @dshchedr  Can I get a test ID to add here ?   this is checking that stalld service is running on a tuned daemon for woker-cnf nodes

Reply:
@yanirq test_id: 35363 please